### PR TITLE
add some more logging to the session

### PR DESCRIPTION
**Pull request by marten-seemann**

### CODE REVIEW

**Comment by marten-seemann** [comment]
@lucas-clemente: Sorry for sending the PR too early. I now fixed all the tests, so you can review now.

File: session.go [hunk 1]
```diff
@@ -684,7 +684,7 @@ func (s *session) logPacket(packet *packedPacket) {
 		return
 	}
 	if utils.Debug() {
-		utils.Debugf("-> Sending packet 0x%x (%d bytes), %s", packet.number, len(packet.raw), packet.encryptionLevel)
+		utils.Debugf("-> Sending packet 0x%x (%d bytes) for connection %x to %s, %s", packet.number, len(packet.raw), s.connectionID, s.conn.RemoteAddr().Network()+" "+s.conn.RemoteAddr().String(), packet.encryptionLevel)
```
Comment (lucas-clemente) [inline session.go]: Can we use the same format `(s -> r)` here as for the "Starting new connection" line above?

File: session.go [hunk 2]
```diff
@@ -769,6 +769,7 @@ func (s *session) scheduleSending() {
 
 func (s *session) tryQueueingUndecryptablePacket(p *receivedPacket) {
 	if s.handshakeComplete {
+		utils.Debugf("Received undecryptable packet from %s after the handshake: %#v, %d bytes data\n", p.remoteAddr.String(), p.publicHeader, len(p.data))
```
Comment (lucas-clemente) [inline session.go]: No need for newlines in log messages.

File: session.go (hunk 1 above)
Comment (marten-seemann) [inline session.go]: I don't think we should log the remote address here at all. I just forgot to remove this. Fixed now.

File: session.go (hunk 2 above)
Comment (marten-seemann) [inline session.go]: Fixed.

**Pull request merged**